CONTRIBUTING.md

```diff
@@ -1,55 +1,156 @@
 
-## Install dependencies
+## Licence and copyright
 
-Go to the `docs/` directory and run:
+Unless explicitly stated in the license headers of source files, all
+contributions to Real-time Ubuntu documentation are licensed under the Creative
+Commons Attribution-Share Alike (CC-BY-SA) 3.0 Unported License.
+To view a copy of this license, visit
+http://creativecommons.org/licenses/by-sa/3.0/ or send a letter to Creative
+Commons, 171 Second Street, Suite 300, San Francisco, California, 94105, USA.
 
-```shell
+All contributors must sign the [Canonical contributor licence agreement],
+which grants Canonical permission to use the contributions. The author of a
+change remains the copyright owner of their code (no copyright assignment
+occurs).
+
+## Releases and versions
+
+Real-time Ubuntu releases align with Ubuntu version updates, with major releases
+occurring twice annually.
+Find the list of releases here: [Real-time Ubuntu release].
+
+## Environment setup
+
+<!-- TODO: Update with your prerequisites or drop if excessive -->
+
+Real-time Ubuntu documentation is built on top of [Canonical's Sphinx starter
+pack] and hosted on [Read the Docs].
+It is written in reStructuredText or MyST Markdown and built with [Sphinx].
+
+- For general guidance, refer to the [starter pack guide]
+- For syntax help and guidelines, refer to the [Canonical style guides]
+- In structuring, the documentation employs the [Diátaxis] approach
+
+### Fork and download the documentation repository
+
+If you are working with this documentation set for the first time, you'll need
+to create a fork of this repository on your GitHub account, and clone the
+forked repository.
+
+### Install dependencies
+
+Go into the `docs/` directory and run:
+
+```
 make install
 ```
 
-This will create a virtual environment with all the tooling dedicated to validate and render the documentation.
-It will also install the [woke snap].
+This will create a virtual environment with all the tooling and packages
+required to validate and render the documentation.
 
-## Build and serve the docs
+### Build and serve the docs
 
 To get a live preview and validation of the documentation, run:
 
-```shell
+```
 make run
 ```
 
-This will provide a locally rendered version of the documentation at: http://localhost:8000
+This will provide a locally rendered version of the documentation at:
+<http://localhost:8000>
 
-The rendered preview will get updated every time a document is modified.
+The rendered preview will get updated every time a document is modified and
+saved.
 
-## Validate the docs
+### Validate the docs
 
-You can validate your changes with the following recipes:
+Before you submit a PR, make sure to validate your changes with the following
+recipes:
 
-```shell
+```
 make spelling  
-make linkcheck 
-make woke      # to check for non-inclusive language
+make linkcheck  
+make woke       # to check for non-inclusive language
```

Comment:
Should there be some guidelines on the wordlists and the sorting?
We can instead add a CI step to check and reject unsorted lists.

Reply:
Hey @farshidtz , what 'sorting' are you referring to here please?

for the wordlist itself it can be found in the [woke repository](https://github.com/get-woke/woke/blob/5d52c1541ab7b5823531dd48144197dda342042e/pkg/rule/default.yaml) itself.

But i'm not sure this is the right place to explain all that other than a quick link to the Woke docs / repo since the woke check does print out alternatives / suggestions.

And we do reference to the starter pack + style guide etc which is where all this general information should be captured.

Reply:
The ordering of the words listed inside the custom wordlist file. I used a tool to sort them a while back and found several duplicates which were hard to notice because entries had been inserted out of order. See https://github.com/canonical/real-time-ubuntu-docs/pull/59

Reply:
I see. that's an easier solve =D
We can add guidelines + a CI step to check.
But let's do that in another PR.